src/components/stack-chart/index.js

```diff
@@ -90,26 +96,33 @@ class StackChartGraph extends React.PureComponent<Props> {
   }
 
   _onSelectedCallNodeChange = (
```

Comment:
Selections appear to be working superficially, but I think it's an incorrect implementation. This function needs to be taught up marker indexes as well. I believe marker index are masquerading as call node indexes here.